### PR TITLE
Added support for labels and existing PR check (v2 from master)

### PR DESCRIPTION
**Pull request by vitalyal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +24,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial release!
-
-[Unreleased]: https://github.com/thomaseizinger/create-pull-request/compare/1.0.0...HEAD
```
Comment (thomaseizinger) [inline CHANGELOG.md:26]: As mentioned in the other PR, this needs to stay.

I'll update the master branch on my repo and then you can rebase / merge.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Unfortunately, not all the checks are passing currently (even ignoring the "reviewers" check): https://github.com/vitalyal/create-pull-request/actions/runs/1058829910

**Comment by vitalyal** [comment]
I've rebased, updated changelog and fixed CI (the only case failing is about reviewers) https://github.com/vitalyal/create-pull-request/actions/runs/1059421040

**Comment by thomaseizinger** [comment]
> I've rebased, updated changelog and fixed CI (the only case failing is about reviewers) https://github.com/vitalyal/create-pull-request/actions/runs/1059421040

Thank you! I'll release a new version soon!

**Comment by thomaseizinger** [comment]
Released as 1.2.0.

**Pull request merged**